lib/user.rb

```diff
@@ -0,0 +1,28 @@
+require './lib/joke'
+require 'pry'
+class User
+attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(insert_joke)
+    @jokes << (insert_joke)
+  end
+
+  def tell(listener, joke)
+    listener.jokes << joke
+  end
```

Comment:
One object should not directly write content into another object. Is there any code we can reuse so Ali knows this joke?